changelog/issue-5699-display-billing-schedule

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: dev
+Comment: NA. This element will be part of a larger changeset. This PR doesn't need a changelog entry
```

Comment:
We've gotta get a label or something that allows us to cleanly silence changelog bot!

Reply:
I've added a retro item. :)